tests/TestRanges.py

```diff
@@ -148,6 +163,132 @@ def testDates(self):
+        self.assertExpectedResult(
+            self.cal.evalRanges("2 days from day after tomorrow", start),
+            (targetStart, targetEnd, 1))
+        self.assertExpectedResult(
+            self.cal.evalRanges("3 days from tomorrow", start),
+            (targetStart, targetEnd, 1))
+        self.assertExpectedResult(
+            self.cal.evalRanges("4 days from today", start),
+            (targetStart, targetEnd, 1))
+
+        targetStart = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec)
+        targetStart = targetStart.timetuple()
+        targetEnd = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec) + datetime.timedelta(days=5)
+        targetEnd = targetEnd.timetuple()
+        self.assertExpectedResult(
+            self.cal.evalRanges("in 5 days", start),
+            (targetStart, targetEnd, 1))
+        self.assertExpectedResult(
+            self.cal.evalRanges("next 5 days", start),
+            (targetStart, targetEnd, 1))
+
+        targetStart = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec) - datetime.timedelta(days=5)
+        targetStart = targetStart.timetuple()
+        targetEnd = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec)
+        targetEnd = targetEnd.timetuple()
+        self.assertExpectedResult(
+            self.cal.evalRanges("since 5 days", start),
+            (targetStart, targetEnd, 1))
+
+        targetStart = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec) - datetime.timedelta(days=1)\
+                      + datetime.timedelta(hours=1)
+        targetStart = targetStart.timetuple()
+        targetEnd = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec)
+        targetEnd = targetEnd.timetuple()
+        self.assertExpectedResult(
+            self.cal.evalRanges("since 23 hours", start),
+            (targetStart, targetEnd, 1))
+
+        targetStart = datetime.datetime(
+            self.yr - 5, self.mth, self.dy, self.hr, self.mn, self.sec)
+        targetStart = targetStart.timetuple()
+        targetEnd = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec)
+        targetEnd = targetEnd.timetuple()
+        self.assertExpectedResult(
+            self.cal.evalRanges("since 5 yrs", start),
+            (targetStart, targetEnd, 1))
+
+        targetStart = datetime.datetime(
+            self.yr , 3, 13, self.hr, self.mn, self.sec)
+        targetStart = targetStart.timetuple()
+        targetEnd = datetime.datetime(
+            self.yr, self.mth, self.dy, self.hr, self.mn, self.sec)
+        targetEnd = targetEnd.timetuple()
+        self.assertExpectedResult(
+            self.cal.evalRanges("since 13th March", start),
+            (targetStart, targetEnd, 1))
+
+        # TEST CASE WORKS, BUT IS RELATIVE
```

Comment:
Please use freezegun library for tests. So that relative test cases can be also be added.
https://pypi.org/project/freezegun/